socorro/external/es/supersearch.py

```diff
@@ -271,93 +299,48 @@ def get(self, **kwargs):
                     if field_data['has_full_version']:
                         name = '%s.full' % name
                     filter_value = param.value
-                elif param.operator == '>':
-                    # greater than
+                elif param.operator in operator_range:
                     filter_type = 'range'
                     filter_value = {
-                        'gt': param.value
-                    }
-                elif param.operator == '<':
-                    # lower than
-                    filter_type = 'range'
-                    filter_value = {
-                        'lt': param.value
-                    }
-                elif param.operator == '>=':
-                    # greater than or equal to
-                    filter_type = 'range'
-                    filter_value = {
-                        'gte': param.value
-                    }
-                elif param.operator == '<=':
-                    # lower than or equal to
-                    filter_type = 'range'
-                    filter_value = {
-                        'lte': param.value
+                        operator_range[param.operator]: param.value
                     }
                 elif param.operator == '__null__':
-                    # is null
                     filter_type = 'missing'
                     args['field'] = name
+                elif param.operator in operator_wildcards:
+                    filter_type = 'query'
+
+                    if field_data['has_full_version']:
+                        name = '%s.full' % name
+
+                    q_args = {}
+                    q_args[name] = (
+                        operator_wildcards[param.operator] % param.value
```

Comment:
Yes, in this case it always is a string, the ``get_parameters`` method made sure of that.